win32ss/CMakeLists.txt

```
@@ -165,6 +165,7 @@ list(APPEND SOURCE
     user/ntuser/winsta.c
     user/ntuser/object.c
     user/rtl/text.c
+    user/rtl/image.c
```

```suggestion
    user/rtl/image.c
    user/rtl/text.c
```
(in alphabetical order here, at least...)